webpack.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = require('carbon-factory/webpack.config')({
+  entryPoint: '/demo/main.js',
```

Comment:
The port for the demo site was `8095`. This has now changed to `8080`

Reply:
updated any reference of 8095 to use 8080